### ISSUE TITLE
Better error message when service name contains capitals

### ISSUE THREAD

**Issue by gtfierro** [immesys/spawnpoint#34]
When deploying a spawnpoint, if the name has capitals in it, you can get a confusing error message e.g.

```
[03/26 09:29:48] sp::TED > [FAILURE] Unable to (re)start container: API error (500): {"message":"Error parsing reference: \"spawnpoint_sp_image_TED\" is not a
 valid repository/tag: repository name must be lowercase"}
```

This is because Docker does not support container names with capitals. It took me a little bit to figure out where the error was coming from (thought it was a misconfiguration). I think having a better error message (or coercing the capitalization to lower case) might help future users.

**Comment by jhkolb** [comment]
Yeah, the error message is definitely unclear, and it would be nice to support service names with capital letters.

This has actually been addressed in version 1.0.

**Comment by gtfierro** [comment]
Cool! I don't mind this being closed, then. I think I'm the largest "legacy spawnpoint" user and I'll be upgrading them all to 1.0 when that comes out

**Comment by jhkolb** [comment]
This has been addressed in the 1.0 release.
